### PR TITLE
Add support for handling multiple deployments

### PR DESCRIPTION
**Pull request by mabuaisha**

### CODE REVIEW

File: pkg/cloudprovider/providers/cloudifyprovider/instances.go [hunk 1]
```diff
@@ -31,6 +32,46 @@ type Instances struct {
 	client     *cloudify.Client
 }
 
+func (r *Instances) GetDeploymentNodeInfo() (map[string] string, error) {
```
Comment (0lvin) [inline pkg/cloudprovider/providers/cloudifyprovider/instances.go]: Add comment for public function.

File: pkg/cloudprovider/providers/cloudifyprovider/instances.go [hunk 2]
```diff
@@ -31,6 +32,46 @@ type Instances struct {
...
+			deploymentInfo["node_data_type"] = dep["node_data_type"].(string)
+			return deploymentInfo, nil
+		}
+	}
+
+	return deploymentInfo, nil
+}
+
+
+func (r *Instances) GetDeploymentNodeID() (map[string]string, error) {
```
Comment (0lvin) [inline pkg/cloudprovider/providers/cloudifyprovider/instances.go]: Add comment for public function.

File: pkg/cloudprovider/providers/cloudifyprovider/loadbalancer.go [hunk 3]
```diff
@@ -29,6 +30,44 @@ type Balancer struct {
 	client     *cloudify.Client
 }
 
+func (r *Balancer) GetDeploymentBalancerInfo() (map[string]string, error) {
```
Comment (0lvin) [inline pkg/cloudprovider/providers/cloudifyprovider/loadbalancer.go]: Add comment for public function.

File: pkg/cloudprovider/providers/cloudifyprovider/loadbalancer.go [hunk 4]
```diff
@@ -29,6 +30,44 @@ type Balancer struct {
...
+			deploymentInfo["id"] = dep["id"].(string)
+			deploymentInfo["node_data_type"] = dep["node_data_type"].(string)
+			return deploymentInfo, nil
+		}
+	}
+
+	return deploymentInfo, nil
+}
+
+func (r *Balancer) GetDeploymentBalancerID() (map[string]string, error) {
```
Comment (0lvin) [inline pkg/cloudprovider/providers/cloudifyprovider/loadbalancer.go]: Add comment for public function.

**Review by 0lvin — CHANGES_REQUESTED** [review]
Looks good, but need to add comments for all public functions(or make private). 

Run "make test" for check.

File: pkg/cloudprovider/providers/cloudifyprovider/instances.go (hunk 1 above)
Comment (mabuaisha) [reply pkg/cloudprovider/providers/cloudifyprovider/instances.go]: Done

File: pkg/cloudprovider/providers/cloudifyprovider/instances.go (hunk 2 above)
Comment (mabuaisha) [reply pkg/cloudprovider/providers/cloudifyprovider/instances.go]: Done

File: pkg/cloudprovider/providers/cloudifyprovider/loadbalancer.go (hunk 3 above)
Comment (mabuaisha) [reply pkg/cloudprovider/providers/cloudifyprovider/loadbalancer.go]: Done

File: pkg/cloudprovider/providers/cloudifyprovider/loadbalancer.go (hunk 4 above)
Comment (mabuaisha) [reply pkg/cloudprovider/providers/cloudifyprovider/loadbalancer.go]: Done

**Review by 0lvin — APPROVED** [review]
Looks good. CR done

**Pull request merged**